doc/man1/srec_input.1

```diff
@@ -540,6 +540,10 @@ of 4 and 8 bytes, the default is 2 bytes.  (Widths in excess of 8 are
 assumed to be number of bits.)
 It is not possible to swap non\[hy]power\[hy]of\[hy]two addresses.
 To change the alignment, use the offset filter before and after.
+.\"             nibble swap
+.TP 8n
+\fB\-Nibble_Swap\fP [ \f[I]width\fP ]
```

Comment:
The width argument doesn't appear to be used. Perhaps a copy-paste error? 

Suggest : `\fB\-Nibble_Swap\fP `

Otherwise, will need to be handled in `srecord/arglex/tool/input.cc`

Reply:
more copypaste, will fix